NORTRIP_multiroad_read_main_inputs.f90

```diff
@@ -866,7 +874,11 @@ subroutine NORTRIP_multiroad_read_receptor_data
                 else             
                     read(unit_in,*,ERR=19) name_receptor(k,1),lon_receptor(k),lat_receptor(k)!,h_receptor(k),name_receptor(k,2)
                     h_receptor(k)=0 !0 height
-                    type_receptor(k)=1 !AQ type
+                    if ( calculation_type == "Avinor" ) then                  
```

Comment:
I would like to discuss the definition of "type_receptor" based on "calculation_type", we once talked about having different receptor types for one simulation?